### DISCUSSION TITLE
Filter result by State or StateCode

### DISCUSSION THREAD

**Discussion by flostony** [komoot/photon#546] [Q&A]
Is there a way to filter the result by state or statecode?
I've tried using OSM tag addr:country or addr:state but all results are empty.

How should I format the url if i want to filter one of the above mentioned tags?

The following url
http://localhost/?q=berlin&osm_tag=addr:country:DE 
doesn't work

**Comment by lonvia** [comment]
That doesn't work at the moment. At least for country there is an open issue (#175).
